build/post-integration-test-docker.sh

```diff
@@ -18,33 +18,26 @@ rm -fr ${it_results} 2>/dev/null
 mkdir -p ${it_results}/server-logs
 mkdir -p ${it_results}/fhir-server-test
 
-containerId=$(docker ps -a | grep fhir-test | cut -d ' ' -f 1)
+containerId=$(docker ps -a | grep fhir | cut -d ' ' -f 1)
 if [[ -z "${containerId}" ]]; then
-    echo "Warning: Could not find fhir-test container!!!"
+    echo "Warning: Could not find fhir container!!!"
 else
-    echo "fhir-test container id: $containerId"
+    echo "fhir container id: $containerId"
 
     # Grab the container's console log
     docker logs $containerId  >& ${it_results}/docker-console.txt
 
     echo "Gathering pre-test server logs from docker container: $containerId"
-    docker cp -L $containerId:/opt/ibm/fhir-server/wlp/usr/servers/fhir-server/logs ${it_results}/server-logs
+    docker cp -L $containerId:/opt/ol/wlp/usr/servers/fhir-server/logs ${it_results}/server-logs
```

Comment:
Why `ol`?

Reply:
read through... it's a-ok now.

Reply:
ol = openliberty, thats where the openliberty image installs liberty